js/src/modals/Verification/verification.js

```diff
@@ -69,11 +75,12 @@ export default class Verification extends Component {
   }
 
   state = {
-    method: 'sms'
+    method: 'sms',
+    store: null
```

Comment:
Why is the store in the state and not on the class itself?

When passed as either a prop or when created on the class it is consistent with what we have elsewhere. Feels really weird having a state in a state. (When done properly, the state should not be on the class using mobx stores at all)